content/en/blog/_posts/2020-05-xx-third-party-dual-sourced-content.md

```diff
@@ -0,0 +1,63 @@
+
+Here's how Kubernetes docs handle third party content now:
+
+> Links to active content in the Kubernetes project (projects in the kubernetes and kubernetes-sigs GitHub orgs) are always allowed.
+>
+> Kubernetes requires some third party content to function. Examples include container runtimes (containerd, CRI-O, Docker), networking policy (CNI plugins), Ingress controllers, and logging.
+>
+> Docs can link to third-party open source software (OSS) outside the Kubernetes project if it’s necessary for Kubernetes to function.
+
+These common-sense guidelines make sure that Kubernetes docs document Kubernetes.
+
+## Keeping the docs focused
+
+Our goal is for Kubernetes docs to be a trustworthy guide to Kubernetes features. To achieve this goal, SIG Docs is [tracking third party content](https://github.com/kubernetes/website/issues/20232) and removing any third party content that isn't: 
+
+- in-project; **and**
+- required for Kubernetes to function.
+
+### Re-homing content
+
+Some content will be removed that readers may find helpful. To make sure readers have continous access to information, we're giving stakeholders untilthe [1.19 release deadline for docs](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.19), **July 9th, 2020** to re-home any content slated for removal.
+
+Over the next few months you'll see less third party content in the docs as contributors open PRs to remove content.
+
+## Background
+
+Over time, SIG Docs observed increasing vendor content in the docs. Some content took the form of vendor-specific implementations that aren't required for Kubernetes to function in-project. Other content was thinly-disguised advertising with minimal to no feature content. Some vendor content was new; other content had been in the docs for years. It became clear that the docs needed clear, well-bounded guidelines for what kind of third party content is and isn't allowed. The [content guide](https://kubernetes.io/docs/contribute/content-guide/) emerged from an extensive period for review and comment from the community.
+
+Docs work best when they're accurate, helpful, trustworthy, and remain focused on features. In our experience, vendor content dilutes trust and accuracy. 
+
+Put simply: feature docs aren't a place for vendors to advertise their products. Our content policy keeps the docs focused on helping developers and cluster admins, not on marketing.
+
+## Ways to contribute
+
+We're tracking third-party content in an [issue in the website repository](https://github.com/kubernetes/website/issues/20232). If you see third party content that's out of project and isn't required for Kubernetes to function, please comment on the tracking issue. 
```

Comment:
nit: ... an issue in the Kubernetes website repository?

Reply:
Any thoughts about moving the `Ways to contribute` after the `dual-sourced` paragraph?
The `contribute` and `want to know more` seem to flow together.

Reply:
> Any thoughts about moving the Ways to contribute after the dual-sourced paragraph?

Good idea!